src/Files.OpenDialog/FilesLauncher/OpenInFolder.cpp

```diff
@@ -73,37 +70,36 @@ LRESULT CALLBACK OpenInFolder::WindowProcedure(HWND hwnd, UINT Msg, WPARAM wPara
 	return DefWindowProc(hwnd, Msg, wParam, lParam);
 }
 
+
 HRESULT OpenInFolder::NotifyShellOfNavigation(PCIDLIST_ABSOLUTE pidl)
```

Comment:
This function has same name and parameters as Explorer++ and has a similar code, so we should remove this as well.
I believe `OpenInFolder` class is only used to make a particular file selected at startup and this file can be deleted completely if we give up this feature.

Reply:
That doesn't matter even though it has same name and arguments.

Reply:
Is it "based on" as defined in the GPL license?

> To “modify” a work means to copy from or adapt all or part of the work in a fashion requiring copyright permission, other than the making of an exact copy. The resulting work is called a “modified version” of the earlier work or a work “based on” the earlier work.

Reply:
- Same function name
- Same function arguemnt
- Different namespace
- Different codes contain

Reply:
Well, it's a delicate point. I'm no expert either, so I'll leave it to your judgment.